internal/test/wmcb/wmcb_test.go

```diff
@@ -297,6 +328,48 @@ func (vm *wmcbVM) runTestKubeletUninstall(t *testing.T) {
 	require.NoError(t, err, "TestKubeletUninstall failed")
 }
 
+// configureContainerd installs the containerd Windows service
+func (vm *wmcbVM) configureContainerd() error {
+	// Check if containerd is running
+	out, err := vm.Run("Get-Process -Name \"containerd\"", true)
+
+	// stderr being empty implies that containerd was running.
+	if err == nil || out == "" {
+		return nil
+	}
+
+	out, err = vm.Run(mkdirCmd(kLog), false)
+	if err != nil {
+		return fmt.Errorf("unable to create remote directory %s: %v\n%s", kLog, err, out)
+	}
+
+	// set Windows defender exclusions for containerd
+	setExclusionCmd := fmt.Sprintf("Add-MpPreference -ExclusionProcess %s", containerdPath)
+	out, err = vm.Run(setExclusionCmd, true)
+	if err != nil {
+		return fmt.Errorf("setting Windows defender process exclusion failed: %v", err)
+	}
+
+	// Start the containerd Windows service
```

Comment:
Starting should also be made into a [shared function](https://github.com/openshift/windows-machine-config-bootstrapper/blob/38466b767564bddeeb68a6a9fab708dd73a937ce/internal/test/wmcb/wmcb_test.go#L237-L245)